lib/libc/zoneinfo/mk_latest_tzrule.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
```

Comment:
mk_latest_tzrule? Do you make latest TZ rule using this script?

Reply:
Remain only the latest rule.